storage/client/queue/README.md

```diff
@@ -108,22 +108,22 @@ The queue service do operations on the queues in the storage account and manage
 
 ### Queue Service Client
 
-The client performs the interactions with the Queue service, create or delete a queue, getting and setting Queue properties, list queues in account, and get queue statistics. An asynchronous, `QueueServiceAsyncClient`, and synchronous, `QueueClient`, client exists in the SDK allowing for selection of a client based on an application's use case.
+The client performs the interactions with the Queue service, create or delete a queue, getting and setting Queue properties, list queues in account, and get queue statistics. An asynchronous, `queueStorageAsyncClient`, and synchronous, `QueueClient`, client exists in the SDK allowing for selection of a client based on an application's use case.
```

Comment:
QueueStorageAsyncClient should be capitalized, it is the name of the class, and QueueClient should be QueueStorageClient

Reply:
Done